pkgs/applications/science/misc/colmap/default.nix

```diff
@@ -1,6 +1,6 @@
 { mkDerivation, lib, fetchFromGitHub, cmake, boost179, ceres-solver, eigen,
   freeimage, glog, libGLU, glew, qtbase,
-  config,
+  config, autoAddDriverRunpath,
```

Comment:
Chances are this will be nixfmt-ed one day, let's make new arguments one per line now